### ISSUE TITLE
allow multiple survey names in inboveg_classification

### ISSUE THREAD

**Issue by hansvancalster** [inbo/inbodb#28]
@ToonHub noticed that the help file of the function `inboveg_classification()` mentions that multiple surveys can be given, but this is currently not possible.  To do: adjust function to allow this in a similar way as has been done in `inboveg_header()`

**Comment by ElsLommelen** [comment]
@hansvancalster There is still an open PR (#27), maybe it is easier for @ElsDeBie if it is merged first?  There are no conflicts to be expected, but it makes the workflow easier.  I don't know why the PR is still open?  For me it is fine to have it closed, and @leymanan also mentions it is OK.  So go ahead unless you have reasons to keep it open.  ;-)

**Comment by ElsDeBie** [comment]
This was already solved in de inbodb:
## get the classification from several specific surveys
#' classif_info <- get_inboveg_classification(con, survey_name = c("MILKLIM_Heischraal2012", "NICHE Vlaanderen" ))

**Comment by ElsLommelen** [comment]
So maybe time to remove the depreciated functions in inborutils to motivate users to start using inbodb?  ;-)
(It has been almost a year since we marked them as depreciated there, I don't know which is a reasonable period to remove functions after marking them as depreciated?)

**Comment by ElsDeBie** [comment]
For me okay to remove them in inborutils. I started writing extra functions (on demand of An Leyssen and Wim Mertens), for from ready (have to dig deep in my memory .. )

**Comment by hansvancalster** [comment]
I merged #27 and will remove the deprecated functions from inborutils.

**Comment by hansvancalster** [comment]
> This was already solved in de inbodb:
> 
> ## get the classification from several specific surveys
> #' classif_info <- get_inboveg_classification(con, survey_name = c("MILKLIM_Heischraal2012", "NICHE Vlaanderen" ))

ok then we can close this issue 👍

**Comment by ElsLommelen** [comment]
At second thought I feel a bit sorry to just remove them completely already, maybe we should just replace the whole content with an error referring to inbodb?  (So functions cannot be used anymore, but people are still informed what to do.)

The reasons why I hesitate to remove it:
- we will get help-desk questions for it, also colleagues that are not involved
- some years ago I used the package development sheetcheat in RStudio, and it was (and still is) outdated: some functions moved to another package years before, and after being depreciated for some years, they were actually removed.  At that point I discovered how hard it is to find the new functions once the reference is removed in the original package...

Maybe wait till the functions are depreciated for one year?
